Calculator/Calculator/CalculateItem.swift

```diff
@@ -0,0 +1,10 @@
+//
+//  CalculateItem.swift
+//  Calculator
+//
+//  Created by Lee minyeol on 2023/10/04.
+//
+
+protocol CalculateItem {
+    
+}
```

Comment:
빈 프로토콜을 잘 작성해주셨네요 👍

중괄호 내에 내용이 없는 경우 간략히 한 줄로 쓰기도 합니다.
```suggestion
protocol CalculateItem { }
```